public_html/wp-content/themes/wporg-events-2023/postcss/base/layout.pcss

```diff
@@ -2,18 +2,6 @@ body {
 	--wp--custom--layout--wide-size: 1600px;
 }
 
-.wp-site-blocks .is-layout-constrained > :where( :not( .alignleft ):not( .alignright ):not( .alignfull ) ),
-.is-root-container.is-layout-constrained > :where( :not( .alignleft ):not( .alignright ):not( .alignfull ) ),
-.block-editor-block-list__layout.is-root-container > :where(:not(.alignleft):not(.alignright):not(.alignfull)),
-.edit-post-visual-editor__post-title-wrapper > :where(:not(.alignleft):not(.alignright):not(.alignfull)) {
-	/*
-	 * `--wp--style--global--content-size` gets set to `180px`, rather than the layout size from the parent's
-	 * `theme.json`. I'm assuming that's a bug somewhere, but haven't tracked it down. This is a hack to make the
-	 * content width usable until that bug is resolved.
-	 */
-	max-width: var( --wp--custom--layout--content-size ) !important;
```

Comment:
This stopped me from using Gutenberg's "content/wide" settings for group blocks which I've used on the organize page to constrain the width.

I could have alternatively set `--wp--custom--layout--content-size: 1120px` in the body above but it isn't clear to me yet that 1120px is the content width for this theme.

After removing this, I didn't see any defects on other pages.